pycrostates/metrics/davies_bouldin.py

```diff
@@ -12,7 +12,7 @@
 
 @fill_doc
 def davies_bouldin_score(cluster):  # lower the better
-    """Compute the Davies-Bouldin score.
+    """Compute the Davies-Bouldin score [1]_.
```

Comment:
That's probably rendering as 'score [1]'.
If you want to remove the space between the word and the reference, you can add a backslash:

'score\ [1]'

Or something like that, I can't remember exactly but I did use this backslash in pycrostates somewhere (search files for a footbibliography) or there is a good example on the sphinx bibtex doc.